test/mustachio/foo.aot_renderers_for_html.dart

```diff
@@ -0,0 +1,83 @@
+// GENERATED CODE. DO NOT EDIT.
+//
+// To change the contents of this library, make changes to the builder source
+// files in the tool/mustachio/ directory.
+
+// Sometimes we enter a new section which triggers creating a new variable, but
+// the variable is not used; generally when the section is checking if a
+// non-bool, non-Iterable field is non-null.
+// ignore_for_file: unused_local_variable
+
+// It is hard to track exact imports without using package:code_builder.
+// ignore_for_file: unused_import
+
+import 'dart:convert' show htmlEscape;
+
+import 'package:dartdoc/dartdoc.dart';
+import 'package:dartdoc/src/generator/template_data.dart';
+import 'foo.dart';
+
+String renderFoo(Foo context0) {
```

Comment:
Random one-off: could/would we pass in a `StringSink` here instead of creating a buffer.

Then you have the option of doing streaming writes later? Maybe?

Reply:
I added a TODO